File: Introductory Problems/repetitions.py
# code by savir singh

string = input()

current = ''
long = 0
longest = 0

for character in string:
    if character == current:
        long += 1
    else:
        if long>longest:
            longest = long
        long = 1
        current = character

if long>longest:
    longest = long

print(longest)

File: Introductory Problems/repetitions.cpp
// code by savir singh

#include <bits/stdc++.h>
using namespace std;

int main()
{
    string str;
    cin >> str;
    char current = ' ';
    long long long1 = 0;
    long long longest = 0;

    for (char character:str) {
        if (character==current) {
            long1+=1;
        }
        else {
            if (long1>longest) {
                longest=long1;
            }
            long1=1;
            current=character;
        }
    }

    if (long1>longest) {
        longest=long1;
    }
    cout << longest;
}
